tests/jsx-text-wrap/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -188,8 +197,9 @@ function DiffOverview(props) {
       <div className="alert alert-info">
         <p>
           This diff overview is computed against the current list of records in
-          this collection and the list it contained on <b>{humanDate(since)}</b>
-          .
+          this collection and the list it contained on <b>
+            {humanDate(since)}
+          </b>.
```

Comment:
This looks like an edge case with the current implementation. When we have tag that would just fit on a line and it is followed by text it causes this unusual formatting.

I imagine in this case we would be aiming for:

```
this collection and the list it contained on 
<b>
  {humanDate(since)}
</b>.
```

I haven't had a chance to investigate why this is happening yet.